frontend/src/components/Map/PanToCurrentLocation.js

```diff
@@ -1,6 +1,7 @@
 import GpsFixedTwoToneIcon from '@material-ui/icons/GpsFixedTwoTone';
 import IconButton from '@material-ui/core/IconButton';
 import styled from "styled-components/macro";
+import React from "react";
```

Comment:
🤔 as of React V17, `React` must not be in scope to use JSX.

Maybe extract the onClick handler to a named function for readability?